Lib/asyncio/locks.py

```diff
@@ -182,10 +182,13 @@ def locked(self):
         This method blocks until the lock is unlocked, then sets it to
         locked and returns True.
         """
-        if not self._locked and all(w.cancelled() for w in self._waiters):
+        if not self._locked and (self._waiters is None or \
```

Comment:
Very minor: please close in brackets all condition `if (test1 and test2)`, drop backslash.